lib/buildsystems/cmake.rb

```diff
@@ -11,6 +11,7 @@ def self.check?(bool = true)
   end
 
   def self.build
+    puts "Additional cmake_options being used: #{@cmake_options}".orange
```

Comment:
```
puts "Additional cmake_options being used: #{@cmake_options}".orange if "#{@cmake_options}" != ""
```
?

Reply:
The point is to show that `meson_options` and `cmake_options` are being seen as empty during the build.